packit_service/config.py

```diff
@@ -58,6 +58,9 @@ def __init__(self):
         self.testing_farm_secret: str = ""
         self.validate_webhooks: bool = True
 
+        # fas.fedoraproject.org needs password to authenticate
+        self.fas_password: Optional[str] = None
+
```

Comment:
Do we need to add also `fas_user` here and into the schema?

Reply:
`fas_user` is already in [`packit.config.Config`](https://github.com/packit-service/packit/blob/master/packit/config/config.py#L52) and [`packit.schema.USER_CONFIG_SCHEMA`](https://github.com/packit-service/packit/blob/master/packit/schema.py#L73). But I do recognise, that seeing this on it's own can be confusing. I'll update the comment, to make this more obvious.